src/lib/data/helpers.ts

```
@@ -0,0 +1,200 @@
+  const now = new UTCDate();
+  const endDate = toDateString(now);
+  let startDate: UTCDate;
+
+  switch (period) {
+    case "daily":
+      // Just today
+      startDate = now;
+      break;
+    case "weekly":
+      // Start of current week (considering Sunday as first day)
+      const day = now.getDay(); // 0 for Sunday, 1 for Monday, etc.
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)